systemtest/src/test/java/io/strimzi/systemtest/CustomResourceStatusST.java

```diff
@@ -170,40 +173,64 @@ void testKafkaBridgeStatus() {
         String connectUrl = "http://my-cluster-connect-api.status-cluster-test.svc:8083";
-        KafkaConnectResource.kafkaConnect(CLUSTER_NAME, 1).done();
+        KafkaConnectResource.kafkaConnect(CLUSTER_NAME, 1)
+            .editMetadata()
+                .addToAnnotations("strimzi.io/use-connector-resources", "true")
+            .endMetadata().done();
         waitForKafkaConnectStatus("Ready");
+        KafkaConnectorResource.kafkaConnector(CLUSTER_NAME, TOPIC_NAME).done();
+        waitForKafkaConnectorStatus(CLUSTER_NAME, "Ready");
         assertKafkaConnectStatus(1, connectUrl);
 
         KafkaConnectResource.replaceKafkaConnectResource(CLUSTER_NAME, kb -> kb.getSpec().setResources(new ResourceRequirementsBuilder()
                 .addToRequests("cpu", new Quantity("100000000m"))
                 .build()));
         waitForKafkaConnectStatus("NotReady");
 
+        KafkaConnectorResource.replaceKafkaConnectorResource(CLUSTER_NAME,
+            kc -> kc.getMetadata().setLabels(Collections.singletonMap("strimzi.io/cluster", "non-existing-connect-cluster")));
+        waitForKafkaConnectorStatus(CLUSTER_NAME, "NotReady");
+
         KafkaConnectResource.replaceKafkaConnectResource(CLUSTER_NAME, kb -> kb.getSpec().setResources(new ResourceRequirementsBuilder()
                 .addToRequests("cpu", new Quantity("10m"))
                 .build()));
         waitForKafkaConnectStatus("Ready");
         assertKafkaConnectStatus(3, connectUrl);
+        KafkaConnectorResource.replaceKafkaConnectorResource(CLUSTER_NAME,
+            kc -> kc.getMetadata().setLabels(Collections.singletonMap("strimzi.io/cluster", CLUSTER_NAME)));
+        waitForKafkaConnectorStatus(CLUSTER_NAME, "Ready");
     }
 
     @Test
     void testKafkaConnectS2IStatus() {
         String connectS2IUrl = "http://my-cluster-s2i-connect-api.status-cluster-test.svc:8083";
         String connectS2IDeploymentConfigName = CONNECTS2I_CLUSTER_NAME + "-connect";
-        KafkaConnectS2IResource.kafkaConnectS2I(CONNECTS2I_CLUSTER_NAME, CLUSTER_NAME, 1).done();
+        KafkaConnectS2IResource.kafkaConnectS2I(CONNECTS2I_CLUSTER_NAME, CLUSTER_NAME, 1)
+            .editMetadata()
+                .addToAnnotations("strimzi.io/use-connector-resources", "true")
+            .endMetadata().done();
         waitForKafkaConnectS2IStatus("Ready");
         assertKafkaConnectS2IStatus(1, connectS2IUrl, connectS2IDeploymentConfigName);
+        KafkaConnectorResource.kafkaConnector(CONNECTS2I_CLUSTER_NAME, TOPIC_NAME).done();
+        waitForKafkaConnectorStatus(CONNECTS2I_CLUSTER_NAME, "Ready");
 
         KafkaConnectS2IResource.replaceConnectS2IResource(CONNECTS2I_CLUSTER_NAME, kb -> kb.getSpec().setResources(new ResourceRequirementsBuilder()
                 .addToRequests("cpu", new Quantity("100000000m"))
                 .build()));
         waitForKafkaConnectS2IStatus("NotReady");
 
+        KafkaConnectorResource.replaceKafkaConnectorResource(CONNECTS2I_CLUSTER_NAME,
```

Comment:
Same as above.

Reply:
Moved to phase when Kafka Connect is ready